whynotearth/BrowTricks
# 196 tenant home

# Issue Being Addressed

#196 

# Type of PR

Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# How to Test/Reproduce

I feel the navigation isn't correct, by adding my account it should be better. right now to reach the tenant home you should log in, then in client list click on back.

# Screenshots/casts

<img width="300" src="https://user-images.githubusercontent.com/510242/90857255-fbf6a180-e398-11ea-8e93-85006d59f157.png" />
